manifests/base/role.yaml

```diff
@@ -101,3 +101,39 @@ rules:
       - delete
     resources:
       - notebooks
+  - verbs:
+      - get
+      - list
+      - watch
+      - create
+      - update
+      - patch
+      - delete
+    apiGroups:
+      - dashboard.opendatahub.io
+    resources:
+      - odhapplications
+  - verbs:
+      - get
+      - list
+      - watch
+      - create
+      - update
+      - patch
+      - delete
+    apiGroups:
+      - dashboard.opendatahub.io
+    resources:
+      - odhdocuments
+  - verbs:
+      - get
+      - list
+      - watch
+      - create
+      - update
+      - patch
+      - delete
```

Comment:
Are we sure we want to grant full access and not just read access?

Reply:
I think read will be fine.

Reply:
FWIW, `watch` could have been in here too... but since we don't use watches today, it probably isn't a concern.